pkg/virt-operator/webhooks/kubevirt-update-admitter_test.go

```diff
@@ -20,112 +20,15 @@
 package webhooks
 
 import (
-	"encoding/json"
-
-	"github.com/golang/mock/gomock"
 	. "github.com/onsi/ginkgo"
 	"github.com/onsi/ginkgo/extensions/table"
 	. "github.com/onsi/gomega"
-	admissionv1 "k8s.io/api/admission/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
 
 	v1 "kubevirt.io/client-go/api/v1"
-	"kubevirt.io/client-go/kubecli"
-	"kubevirt.io/kubevirt/pkg/virt-api/webhooks"
 )
 
 var _ = Describe("Validating KubeVirtUpdate Admitter", func() {
 
-	getAdmitter := func(needsVMIMock bool) *KubeVirtUpdateAdmitter {
-		ctrl := gomock.NewController(GinkgoT())
-		virtClient := kubecli.NewMockKubevirtClient(ctrl)
-
-		if needsVMIMock {
-			vmiInterface := kubecli.NewMockVirtualMachineInstanceInterface(ctrl)
-			virtClient.EXPECT().VirtualMachineInstance(gomock.Any()).Return(vmiInterface).AnyTimes()
-		}
-
-		return NewKubeVirtUpdateAdmitter(virtClient)
-	}
-
-	getKV := func() v1.KubeVirt {
-		return v1.KubeVirt{
-			ObjectMeta: metav1.ObjectMeta{
-				Namespace: "default",
-			},
-			Spec: v1.KubeVirtSpec{
-				Workloads: nil,
-			},
-		}
-	}
-
-	getComponentConfig := func() v1.ComponentConfig {
-		return v1.ComponentConfig{
-			NodePlacement: &v1.NodePlacement{
-				NodeSelector: map[string]string{
-					"kubernetes.io/hostname": "node01",
-				},
-			},
-		}
-	}
-
-	It("should accept workload update when no VMIS are running", func() {
-		kvAdmitter := getAdmitter(true)
-		kv := getKV()
-		kvBytes, _ := json.Marshal(&kv)
-
-		cc := getComponentConfig()
-		kv.Spec.Workloads = &cc
-		kvUpdateBytes, _ := json.Marshal(&kv)
-
-		ar := &admissionv1.AdmissionReview{
-			Request: &admissionv1.AdmissionRequest{
-				Resource: webhooks.KubeVirtGroupVersionResource,
-				Object: runtime.RawExtension{
-					Raw: kvUpdateBytes,
-				},
-				OldObject: runtime.RawExtension{
-					Raw: kvBytes,
-				},
-				Operation: admissionv1.Update,
-			},
-		}
-
-		resp := kvAdmitter.Admit(ar)
-		Expect(resp.Allowed).To(BeTrue())
-	})
```

Comment:
Why was this (and others) test case removed?

Reply:
@zcahana I've decided to address this in a separate PR: https://github.com/kubevirt/kubevirt/pull/6120

Reply:
Not sure I entirely follow... do you intent to remove these tests in #6120 or in this PR?

Reply:
@zcahana Sorry for the confusion. Yes, I would remove them in #6120 and will rebase this PR on it.

Reply:
@zcahana Merged